cmd/backup-manager/app/util/util.go

```diff
@@ -353,18 +355,37 @@ func GetBRMetaData(ctx context.Context, provider v1alpha1.StorageProvider) (*kvb
 		return nil, err
 	}
 	defer s.Close()
-	exist, err := s.Exists(ctx, constants.MetaFile)
-	if err != nil {
-		return nil, err
-	}
-	if !exist {
-		return nil, fmt.Errorf("%s not exist", constants.MetaFile)
 
+	var metaData []byte
+	// use exponential backoff, every retry duration is duration * factor ^ (used_step - 1)
+	backoff := wait.Backoff{
+		Duration: time.Second,
+		Steps:    6,
+		Factor:   2.0,
+		Cap:      time.Minute,
+	}
+	readBackupMeta := func() error {
+		exist, err := s.Exists(ctx, constants.MetaFile)
+		if err != nil {
+			return err
```

Comment:
why not retry for this error too? It could be some other temp errors.

Reply:
All the errors in `readBackupMeta` function can retry except the metadata file not existed.